src/pages/DataAccessRequestApplication.js

```diff
@@ -81,7 +84,11 @@ class DataAccessRequestApplication extends Component {
         cloudUse: '',
         cloudProvider: '',
         cloudProviderType: '',
-        cloudProviderDescription: ''
+        cloudProviderDescription: '',
+        gsoAcknowledgement: false,
+        pubAcknowledgement: false,
+        dsAcknowledgement: false,
+        activeDULQuestions: {}
```

Comment:
Can you add a story for adding these to the back end? I'd love it if you could take that on too. Here are a couple PRs that do very similar things for additional string/boolean fields as well as an object based on for collaborators:
* https://github.com/DataBiosphere/consent/pull/704
* https://github.com/DataBiosphere/consent/pull/778